launcher/ui/themes/BrightTheme.cpp

```diff
@@ -18,19 +18,19 @@ bool BrightTheme::hasColorScheme()
 QPalette BrightTheme::colorScheme()
 {
     QPalette brightPalette;
-    brightPalette.setColor(QPalette::Window, QColor(239,240,241));
-    brightPalette.setColor(QPalette::WindowText, QColor(49,54,59));
-    brightPalette.setColor(QPalette::Base, QColor(252,252,252));
-    brightPalette.setColor(QPalette::AlternateBase, QColor(239,240,241));
-    brightPalette.setColor(QPalette::ToolTipBase, QColor(49,54,59));
-    brightPalette.setColor(QPalette::ToolTipText, QColor(239,240,241));
-    brightPalette.setColor(QPalette::Text,  QColor(49,54,59));
-    brightPalette.setColor(QPalette::Button, QColor(239,240,241));
-    brightPalette.setColor(QPalette::ButtonText, QColor(49,54,59));
+    brightPalette.setColor(QPalette::Window, QColor(255,255,255));
+    brightPalette.setColor(QPalette::WindowText, QColor(17,17,17));
+    brightPalette.setColor(QPalette::Base, QColor(250,250,250));
+    brightPalette.setColor(QPalette::AlternateBase, QColor(246,246,246));
```

Comment:
i feel like this is too bright for an alternative base, i can barely make out the difference between them in a list...
before: 
![image](https://user-images.githubusercontent.com/9145768/193422998-0c5d9531-5216-4c98-98e0-b54a72055a88.png)
after:
![image](https://user-images.githubusercontent.com/9145768/193423013-4fe1cd4d-a4ff-4b03-9b2e-fe36fb2a6fe1.png)